virttest/utils_conn.py

```diff
@@ -1230,8 +1269,12 @@ def server_setup(self, on_local=False):
 
         # edit the /etc/libvirt/libvirtd.conf to add tls_port=$tls_port
         if tls_port != '16514':
-            pattern_to_repl = {r".*tls_port\s*=\s*.*": 'tls_port="%s"' % tls_port}
-            operate_libvirtdconf.sub_else_add(pattern_to_repl)
```

Comment:
maybe can be simplified as:
pattern_ro_repl=xx
if libvirt_version(5,6):
     pattern_ro_reply=yy
operate_libvirtdconf_sun_lese_add(pattern_ro_reply)

Reply:
need to update different pattern in different configuration file here.